packages/vite/src/node/optimizer/index.ts

```diff
@@ -787,8 +741,7 @@ const KNOWN_INTEROP_IDS = new Set(['moment'])
 function needsInterop(
   id: string,
   exportsData: ExportsData,
-  outputs: Record<string, any>,
-  cacheDirOutputPath: string
+  output: any
```

Comment:
Could we somehow type this?
E.g. with at least `output: { exports: string[] }`

Alternative, if we only have exports as single thing in the future, then we may just use `exports: string[]` (will not directly work in this way, cause variable conflicts)